src/explore-education-statistics-frontend/src/modules/api/permalink/__tests__/createPermalinkTable.test.ts

```diff
@@ -1,5 +1,6 @@
 import createPermalinkTable from '@frontend/modules/api/permalink/createPermalinkTable';
 import { createApiMocks } from '@frontend-test/apiRouteMocks';
+import { Footnote } from '@common/services/types/footnotes';
```

Comment:
I don't think we need this import anymore.

Reply:
Good spot, I've removed it now.